### ISSUE TITLE
pkgconfig entry

### ISSUE THREAD

**Issue by alex-tee** [sadko4u/lsp-dsp-lib#3]
is there a reason no pkgconfig entry is installed?

**Comment by sadko4u** [comment]
Probably because I'm not very familiar with pkg-config files.
Anyway, it would be a good and proper solution to generate and install the '.pc' file.

**Comment by sadko4u** [comment]
I'll add '*.pc' file generation.

**Comment by alex-tee** [comment]
```pkgconfig
prefix=<PREFIX HERE>
libdir=${prefix}/lib
includedir=${prefix}/include

Name: lsp-dsp-lib
Description: Digital signal processing library.
Version: 0.5.4
Libs: -L${libdir} -llsp-dsp-lib
Cflags: -I${includedir}
```

something like this would do

**Comment by alex-tee** [comment]
that should go to $DESTDIR$PREFIX/lib/pkgconfig/lsp-dsp-lib.pc

**Comment by sadko4u** [comment]
Pushed changes to the **devel** branch, please verify: 5698e2a23a2a80c11375b069c363b97d415f9473

**Comment by alex-tee** [comment]
it works, thanks

this works with meson now! `lsp_dsp_dep = dependency ('lsp-dsp-lib')`

`Run-time dependency lsp-dsp-lib found: YES 0.5.4`

**Comment by sadko4u** [comment]
Merged changes to **master**: 946a41e85d9f3ce92a6c1565e4c17395f2fe614c
